theme/less/_navigation.less

```diff
@@ -1,25 +1,132 @@
-nav
+// The main navigation menu
+
+#main-navigation 
 {
-  ul
+  background: linear-gradient(#fcfcfc, #ececec);
+}
+
+#main-navigation > ul
+{
+  display: flex;
+  padding: 0;
+  border: #F5F5F5 solid  1px;
+  border-radius: 5px;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.2);
+}
+
+#main-navigation > ul li
+{ 
+  display: flex;
+  justify-content:space-between;
+  position: relative;
+  list-style: none;
+  border-right: 1px solid #e3e3e3;   
+}
+
+#main-navigation ul ul li
+{
+   min-width: 150px;
+}
+
+#main-navigation > ul > li:hover
+{
+  background-color: #f0f0f0;
+}
+
+#main-navigation ul ul li:hover, ul ul a:hover
+{
+  background-color: #ffffff;
+}
+
+#main-navigation ul a
+{
+  padding: 15px;
+  flex-grow: 2;
+}
+
+#main-navigation > ul ul
+{
+  top: 48px;
+  display: block;
+  position: absolute;
+  left: 0;
+  padding: 0;
+  background-color: #f0f0f0;
+}
+
+#main-navigation > ul ul ul
+{
+  top: 0;
+  left: 99%;
+  margin-left: 1px;
+}
+
+#main-navigation > ul li:not(:hover) > ul
+{
+  display: none;
+}
+
+#main-navigation ul ul a
+{
+  color: #595959;
+}
+
+#main-navigation > ul > li > a
+{ padding: 15px 10px;
+  font-weight: bold;
+  font-size: 14px;
+  color:#2883DE;
+  text-shadow: 1px 1px 0px #fff;
+}
+
+#main-navigation > ul > li:hover a
+{
+color: #545454;
+}
+
+#main-navigation > ul > li.has-children > a:after
+{
+  content: url('/images/arrow_bullet.png');
+  padding-left: 10px;
+}
+
+#main-navigation ul ul li.has-children > a:after
+{
+  content: url('/images/arrow_bullet.png');
+  position: absolute;
+  right: 10px;
+}
+
+#main-navigation > ul > li.has-children > a:hover:after
+{
+  content: url('/images/arrow_bullet_dark.png');
+  padding-left: 10px;
+}
+
+#main-navigation ul ul li.has-children > a:hover:after
+{
+  content: url('/images/arrow_bullet_dark.png');
```

Comment:
I'm impressed by this! Good job. Didn't know that it's possible to use `url()` in content.